src/engine/QueryPlanner.cpp

```diff
@@ -22,21 +23,116 @@
 #include "Union.h"
 
 // _____________________________________________________________________________
-QueryPlanner::QueryPlanner(QueryExecutionContext* qec) : _qec(qec) {}
+QueryPlanner::QueryPlanner(QueryExecutionContext* qec)
+    : _qec(qec), _randVarCount(0) {
+  _randomSeed = time(NULL);
```

Comment:
Do we want to set this to something static to make runs comparable?

Reply:
Seems like a good idea. I've set the seed to 42.